pkgs/by-name/nc/ncbi-amrfinderplus/package.nix

```
@@ -0,0 +1,78 @@
+{
+  blast,
+  curl,
+  fetchFromGitHub,
+  hmmer,
+  lib,
+  makeWrapper,
+  stdenv,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ncbi-amrfinderplus";
+  version = "4.0.23";
+
+  src = fetchFromGitHub {
+    owner = "ncbi";
+    repo = "amr";
+    tag = "amrfinder_v${version}";
+    hash = "sha256-Rnw83v19dc9Hi/S6MqdYqz/kbTUWrp5i8gpaaPsDK1A=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ curl ];
+
+  nativeBuildInputs = [ makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeBinaryWrapper ];
```

Nit: `makeBinaryWrapper` is preferred to `makeWrapper`